lib/plugins/login/login.js

```diff
@@ -0,0 +1,132 @@
+'use strict';
+
+const BbPromise = require('bluebird');
+const crypto = require('crypto');
+const readline = require('readline');
+const opn = require('opn');
+const fetch = require('node-fetch');
+const jwtDecode = require('jwt-decode');
+const chalk = require('chalk');
+const getFrameworkId = require('../../utils/getFrameworkId');
+const updateConfig = require('../../utils/config/update');
+
+const config = {
```

Comment:
Here is okay for now. We can move to centralized spot once more default config stuff arises